tests/cupy_tests/core_tests/test_dlpack.py

```diff
@@ -6,25 +6,42 @@
 from cupy import testing
 
 
+def _gen_array(dtype):
+    if cupy.issubdtype(dtype, cupy.unsignedinteger):
+        array = cupy.random.randint(
+            0, 10, size=(2, 3)).astype(dtype)
+    elif cupy.issubdtype(dtype, cupy.integer):
+        array = cupy.random.randint(
+            -10, 10, size=(2, 3)).astype(dtype)
+    elif cupy.issubdtype(dtype, cupy.floating):
+        array = cupy.random.rand(
+            2, 3).astype(dtype)
+    elif cupy.issubdtype(dtype, cupy.complexfloating):
+        array = cupy.random.random((2,3)).astype(dtype)
```

Comment:
This actually fixes a test bug (complex dtypes not covered) which I will backport manually.

Reply:
See #5425.